examples/package.json

```diff
@@ -6,6 +6,6 @@
 	"dependencies": {
 		"glfw-raub": "https://github.com/node-3d/glfw-raub.git",
 		"image-raub": "https://github.com/node-3d/image-raub.git",
-		"webgl-raub": "https://github.com/node-3d/webgl-raub.git"
+		"webgl-raub": "1.3.0"
```

Comment:
but why? also what about glfw and image?

Reply:
heh apparently my find/replace didn't catch these :-]